pkg/agent/agent.go

```diff
@@ -229,17 +252,22 @@ func (a *Agent) RemoveHostResources(ctx context.Context, kubeClient client.Clien
 
 // Reconcile reconciles the environment and target on the landscaper.
 func (a *Agent) Reconcile(ctx context.Context, req reconcile.Request) (reconcile.Result, error) {
+	logger, ctx := logging.MustStartReconcileFromContext(ctx, req, nil)
 	if req.Name != a.config.Name {
+		logger.Info("Not responsible for this environment", "targetEnvironment", a.config.Name)
```

Comment:
Do we need a constant for targetEnvironment?

Reply:
Probably not, I don't think this key will appear more than once. Hm, maybe we don't even need it here, I think I add the name of the environment the agent is responsible for already to the values at a different point. Will check.